pkgs/tools/virtualization/multipass/default.nix

```diff
@@ -0,0 +1,128 @@
+{ cmake
+, dnsmasq
+, fetchFromGitHub
+, git
+, gtest
+, iproute2
+, iptables
+, lib
+, libapparmor
+, libvirt
+, libxml2
+, nixosTests
+, openssl
+, OVMF
+, pkg-config
+, qemu
+, qemu-utils
+, qtbase
+, qtx11extras
+, slang
+, stdenv
+, wrapQtAppsHook
+, xterm
+}:
+
+let
+  pname = "multipass";
+  version = "1.11.0";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "canonical";
+    repo = "multipass";
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-2d8piIIecoSI3BfOgAVlXl5P2UYDaNlxUgHXWbnSdkg=";
+    fetchSubmodules = true;
```

Comment:
Using submodules is problematic because eg we are using a two year old fork of libssh https://github.com/canonical/multipass/blob/main/.gitmodules#L9-L11

Reply:
Please, can you open an issue documenting your remarks post-merge?

Reply:
As this is directly related to this PR, I think that is unnecessary.